encoder/vaapiencoder_base.cpp

```diff
@@ -89,12 +89,12 @@ YamiStatus VaapiEncoderBase::start(void)
 void VaapiEncoderBase::flush(void)
 {
     AutoLock l(m_lock);
-    m_output.clear();
 }
 
 YamiStatus VaapiEncoderBase::stop(void)
 {
     FUNC_ENTER();
+    m_output.clear();
     cleanupVA();
```

Comment:
Then why is it needed here?  h.264 stop is calling flush, so the problem might be that your changes to h.264 flush don't need to call  base class flush